tests/gold_tests/pluginTest/cert_update/cert_update.test.py

```diff
@@ -75,10 +75,6 @@
 # Set up plugin
 Test.PreparePlugin(Test.Variables.AtsExampleDir + '/plugins/c-api/cert_update/cert_update.cc', ts)
 
-ts.Disk.plugin_config.AddLine(
```

Comment:
Probably want to pull these unrelated test cleanups to a separate PR.

Reply:
Sounds good. I reverted these other test changes from this PR.

Reply:
Added a new PR for the test tweaks here:

https://github.com/apache/trafficserver/pull/6069